Makefile

```diff
@@ -37,6 +37,12 @@ clean-sa:
 clean-la:
 	rm -f $(BIN) 
 
+.PHONY: install-tools
+install-tools:
+	go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen
+	go install google.golang.org/protobuf/cmd/protoc-gen-go
+	go install go.uber.org/mock/mockgen
```

Comment:
to make the build more stable, don’t you want to declare a given version?

Reply:
Thanks for taking a look at this, @thomas-fossati.

I thought about the same, and found out that go install respects the version specified in go.mod. I didn't want to specifiy the version in both places as it'd be harder to maintain if someone updates go.mod.

Let me see if I can find any documentation about this, and get back. I'll also run some tests and confirm. Thank you so much!